test/framework/src/main/java/org/elasticsearch/test/TestCluster.java

```diff
@@ -127,46 +127,40 @@ private void deleteTemplates(Set<String> excludeTemplates, ActionListener<Void>
             l -> client().execute(GetComponentTemplateAction.INSTANCE, new GetComponentTemplateAction.Request(TEST_REQUEST_TIMEOUT, "*"), l)
         );
 
-        SubscribableListener
-
-            // dummy start step for symmetry
-            .newSucceeded(null)
-
-            // delete composable templates
-            .<GetComposableIndexTemplateAction.Response>andThen(getComposableTemplates::addListener)
-            .<AcknowledgedResponse>andThen((l, r) -> {
-                var templates = r.indexTemplates()
-                    .keySet()
-                    .stream()
-                    .filter(template -> excludeTemplates.contains(template) == false)
-                    .toArray(String[]::new);
-                if (templates.length == 0) {
-                    l.onResponse(AcknowledgedResponse.TRUE);
-                } else {
-                    var request = new TransportDeleteComposableIndexTemplateAction.Request(templates);
-                    client().execute(TransportDeleteComposableIndexTemplateAction.TYPE, request, l);
-                }
-            })
-            .andThenAccept(ElasticsearchAssertions::assertAcked)
-
-            // then delete component templates
-            .<GetComponentTemplateAction.Response>andThen(getComponentTemplates::addListener)
-            .<AcknowledgedResponse>andThen((l, response) -> {
-                var componentTemplates = response.getComponentTemplates()
-                    .keySet()
-                    .stream()
-                    .filter(template -> excludeTemplates.contains(template) == false)
-                    .toArray(String[]::new);
-                if (componentTemplates.length == 0) {
-                    l.onResponse(AcknowledgedResponse.TRUE);
-                } else {
-                    client().execute(
-                        TransportDeleteComponentTemplateAction.TYPE,
-                        new TransportDeleteComponentTemplateAction.Request(componentTemplates),
-                        l
-                    );
-                }
-            })
+        // dummy start step for symmetry
+        SubscribableListener.nullSuccess()
```

Comment:
I would rather reinstate the whitespace here with a blank line after the `SubscribableListener`. I know Spotless should really know how to line all the `.` operators up but for some reason it doesn't without that line.

Reply:
Done :)